editor.planx.uk/src/@planx/components/shared/PrivateFileUpload/Dropzone.tsx

```diff
@@ -27,10 +26,11 @@ const FauxLink = styled(Box)(({ theme }) => ({
   whiteSpace: "nowrap",
 }));
 
-const Root = styled(ButtonBase, {
+const Root = styled(Box, {
```

Comment:
When this was a `button`, I was hitting this a11y issue - https://dequeuniversity.com/rules/axe/4.9/nested-interactive

Using a `div` here instead of a `button` matches[ the suggested usage in the docs](https://react-dropzone.js.org/#section-basic-example).